tests/unit/ligo-plugin-tests/ligo-plugin-tests.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { ___TEST___ } from '../../../taqueria-plugin-ligo/compile';
+
+describe('testing Ligo compile.ts getContractNameForExpr', () => {
+	type ExprKind = 'storage' | 'default_storage' | 'parameter';
```

Comment:
This should be exported via __TEST__ instead of being redefined

Reply:
you mean ExprKind ?

ok

Reply:
it works, cool, ok done

Reply:
yes ExprKind